src/building-blocks/FormField.tsx

```diff
@@ -48,15 +49,20 @@ class FormField extends Component<IFormFieldProps> {
       return null;
     }
 
-    const { formManager, formModel } = this.props,
+    const { formManager, formModel, ...passDownProps } = this.props,
       { skipFieldDecorator, editComponent: EditComponent } = this.fieldConfig;
 
     if (skipFieldDecorator) {
       return <EditComponent {...this.editProps} />;
     }
 
     return (
-      <FormItem fieldConfig={this.fieldConfig} formManager={formManager} formModel={formModel}>
+      <FormItem
+        fieldConfig={this.fieldConfig}
+        formManager={formManager}
+        formModel={formModel}
+        {...omit(passDownProps, ['fieldConfig'])}
```

Comment:
if you move `{...passDownProps}` to the top of this list, `fieldConfig={` will override it. I can't decide if that's better because we don't use omit or worse because it breaks on sort and is less verbose. Your call. Maybe put fieldconfig at the bottom with a note of we decide to go that way, or just keep it like this.